### ISSUE TITLE
Translation bug in contextualmenu.

### ISSUE THREAD

**Issue by zer0kool** [Delni/mercury#9]
Note: I am not a GitHub guru so I really don't know how all of this works other than a place to store your code. 

There is a bug on the contextMenu function in file HTMLEventHandler.js that makes the recurrence "timespan" unable to be translated. 

If anyone is translating then you will need to replace one line in HTMLEventHandler.js 
From:
`contextualMenu.append(new MenuItem({label: i18njs('Recurrence: every %n ',resSQL.offset)+i18njs(resSQL.timespan,resSQL.offset), enabled: false}))`
To:
`contextualMenu.append(new MenuItem({label: i18njs('Recurrence every %n ',resSQL.offset)+i18njs(resSQL.timespan), enabled: false}))`

Then add the following values to your lang file es.json if you're translating into Spanish.
    "days" : "dias"
    "weeks" : "semanas",
    "months" : "meses",
    "quarters" : "trimestres",
    "years" : "años",

I am cloning the repo to build a project for the community if that is not a problem for you.

**Comment by Delni** [comment]
Seeing from the code you show, it seems that you are working on v1 code. I have fully refactored the code and therefore I do not use the same i18n engine anymore. (v2 is still on going, and not ready for release)

You are more than welcome to clone the repo 😄 this is why I've made it open sourced 👍

**Comment by zer0kool** [comment]
You're building v2 on  Vue.js correct?
Thank you, Delni.

**Comment by Delni** [comment]
Correct. I'm building v2 on top of [electron-vue](https://github.com/SimulatedGREG/electron-vue). I hope it will result better scability and maintainibility. In v1 some html was generated from js, which was awful to edit.
